src/transformers/tools/agents.py

```diff
@@ -487,7 +497,7 @@ def generate_one(self, prompt, stop):
 
         response = requests.post(self.url_endpoint, json=inputs, headers=headers)
         if response.status_code == 429:
-            print("Getting rate-limited, waiting a tiny bit before trying again.")
+            logger.info("Getting rate-limited, waiting a tiny bit before trying again.")
```

Comment:
What's the reason for this one to be `logger` rather than `self.log`?

Reply:
It's not really a result message compared to the other, more like an info of what's taking time. It should have been `logger.info` before and not `print` IMO.